.circleci/config.yml

```diff
@@ -27,9 +27,20 @@ jobs:
       - run:
           name: Run tests
           command: yarn prettier --check 'src/**/*'
+  check-eslint:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: yarn
```

Comment:
We should look into if we can cache these installs to speed up CI runs. For a follow-up in any event.

Reply:
Have a look at https://github.com/canonical-web-and-design/maas-ui/blob/master/.circleci/config.yml for an example of how to cache deps.

Reply:
Nice - I've just added this.